ts/screens/authentication/UnrecognizedCie.tsx

```diff
@@ -0,0 +1,70 @@
+import { Container, H1, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+type Props = OwnProps;
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: customVariables.contentPadding
+  },
+  text: {
+    fontSize: customVariables.fontSizeBase
+  }
+});
+
+class UnrecognizedCie extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render(): React.ReactNode {
+    const cancelButtonProps = {
+      block: true,
+      cancel: true,
+      onPress: (): boolean => this.props.navigation.goBack(),
```

Comment:
```suggestion
     onPress: (): void => {
        this.props.navigation.goBack();
      },
```
onPress requires `onPress?: () => void;`